tala_connector.py

```diff
@@ -597,8 +606,12 @@ def _handle_download_policy_bundle(self, param):
 
         project_id = param['project_ids']
         try:
-            [int(x) for x in project_id.split(",")]
-        except:
+            project_id_formatted = []
+            for x in project_id.split(","):
+                if int(x) < 0:
```

Comment:
Same as above.

Reply:
Reverted back as I do not get a proper document about project id validation.